src/Controls/src/Core/HybridWebView/HybridWebView.cs

```diff
@@ -15,40 +25,215 @@ public class HybridWebView : View, IHybridWebView
 			BindableProperty.Create(nameof(HybridRoot), typeof(string), typeof(HybridWebView), defaultValue: "wwwroot");
 
 
-		/// <summary>
-		/// Specifies the file within the <see cref="HybridRoot"/> that should be served as the default file. The
-		/// default value is <c>index.html</c>.
-		/// </summary>
+		/// <inheritdoc/>
 		public string? DefaultFile
 		{
 			get { return (string)GetValue(DefaultFileProperty); }
 			set { SetValue(DefaultFileProperty, value); }
 		}
 
-		/// <summary>
-		///  The path within the app's "Raw" asset resources that contain the web app's contents. For example, if the
-		///  files are located in <c>[ProjectFolder]/Resources/Raw/hybrid_root</c>, then set this property to "hybrid_root".
-		///  The default value is <c>wwwroot</c>, which maps to <c>[ProjectFolder]/Resources/Raw/wwwroot</c>.
-		/// </summary>
+		/// <inheritdoc/>
 		public string? HybridRoot
 		{
 			get { return (string)GetValue(HybridRootProperty); }
 			set { SetValue(HybridRootProperty, value); }
 		}
 
-		void IHybridWebView.RawMessageReceived(string rawMessage)
+		void IHybridWebView.MessageReceived(string rawMessage)
```

Comment:
This processing should be done in Core and fire specific methods in Controls. There is no need for XAML objects to be post processing and parsing strings.